src/protocolsupport/protocol/typeremapper/itemstack/LegacyItemType.java

```diff
@@ -55,6 +64,7 @@ public void applyDefaultRemaps() {
 			registerRemapEntry(Material.SCUTE, Material.LIME_DYE, ProtocolVersionsHelper.BEFORE_1_13);
 			registerRemapEntry(Material.TURTLE_HELMET, Material.CHAINMAIL_HELMET, ProtocolVersionsHelper.BEFORE_1_13);
 			registerRemapEntry(Material.TRIDENT, Material.DIAMOND_HOE, ProtocolVersionsHelper.BEFORE_1_13);
+			registerRemapEntry(Material.DEBUG_STICK, Material.STICK, ProtocolVersionsHelper.BEFORE_1_13);
```

Comment:
What's a DEBUG_STICK? :-o

Reply:
[This](https://minecraft.gamepedia.com/Debug_Stick)